app/controllers/api/v1/pages_controller.rb

```diff
@@ -100,4 +114,22 @@ def not_found
   def missing_parameter(exception)
     render json: { error: exception.message }, status: :bad_request
   end
+
+  def convert_old_answer_types_to_new_format(page)
```

Comment:
could we rename `page` as we already have a private method in this controller thats called `page` which makes its confusing to follow and understand that we are modifying the object thats passed to this method rather the `@page` which the page method would have set.

Reply:
Done - called it `page_object` to make clear that's what it's expecting